examples/subscriptions/package.json

```diff
@@ -11,6 +11,39 @@
     "codegen": "shopify hydrogen codegen"
   },
   "dependencies": {
-    "@shopify/cli-hydrogen": "*"
-  }
+    "@remix-run/react": "^2.9.2",
+    "@shopify/cli": "3.61.2",
+    "@shopify/cli-hydrogen": "^8.1.0",
+    "@shopify/hydrogen": "2024.4.3",
+    "@shopify/remix-oxygen": "^2.0.4",
+    "graphql": "^16.6.0",
+    "graphql-tag": "^2.12.6",
+    "isbot": "^3.6.6",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "^2.9.2",
+    "@remix-run/eslint-config": "^2.9.2",
+    "@shopify/mini-oxygen": "^3.0.3",
+    "@shopify/oxygen-workers-types": "^4.0.0",
+    "@shopify/prettier-config": "^1.1.2",
+    "@total-typescript/ts-reset": "^0.4.2",
+    "@types/eslint": "^8.4.10",
+    "@types/react": "^18.2.22",
+    "@types/react-dom": "^18.2.7",
+    "eslint": "^8.20.0",
+    "eslint-plugin-hydrogen": "0.12.2",
+    "postcss": "^8.4.21",
+    "postcss-import": "^15.1.0",
+    "postcss-preset-env": "^8.2.0",
+    "prettier": "^2.8.4",
+    "typescript": "^5.2.2"
+  },
```

Comment:
This is because the branch is not updated. I had to remove tailwind@v3 from this example but it has now been turned into a diff, so we can remove this part.